Makefile

```diff
@@ -0,0 +1,6 @@
+lint:
+# ignore errors for long lines and multi-statement lines
+	@pipx run flake8 --ignore=E501,E701,W503 .
```

Comment:
How is `flake8` getting installed?  Same for `rstcheck` below.

Modern projects use `pre-commit` instead of `make`.